### PR TITLE
fixes download/addons and donation pages + layouts

### PR DESCRIPTION
**Pull request by veroandreo**
I created a specific layout for AddOns to avoid getting the "GRASS GIS for XX" title that makes sense for different OS but not for addons. 
The PR also fixes img sizing (I changed MD style to HTML tags to be able to control size).  
And I also fixed the logos' path for download landing page.

### CODE REVIEW

File: content/download/addons.en.md [hunk 1]
```diff
@@ -4,27 +4,28 @@ date: 2018-12-29T11:02:05+06:00
 description: "Download GRASS GIS user modules"
```
Comment (neteler) [inline content/download/addons.en.md:4]:
```suggestion
description: "Download GRASS GIS addon extension modules"
```

(maybe without "extension"? It should be consistent ...

File: content/download/addons.en.md [hunk 2]
```diff
@@ -4,27 +4,28 @@ date: 2018-12-29T11:02:05+06:00
...
-You can also use the command line
+You can also use the command line, for example:
 
     g.extension extension=r.fuzzy.system
 
 
 ### <a name="Power-user"></a>Power users/developers
 
-The AddOns source code is hosted in [GRASS-AddOns Github repository](https://github.com/osgeo/grass-addons). To manually compile see the [compile and install wiki page](https://grasswiki.osgeo.org/wiki/Compile_and_Install#Addons). 
+The AddOns source code is also hosted in <a href="https://github.com/OSGeo/grass-addons" target="_blank"><i class="fa fa-github"></i> Github</a>:
```
Comment (neteler) [inline content/download/addons.en.md:28]: AddOns --> addons?

File: content/download/addons.en.md [hunk 3]
```diff
@@ -4,27 +4,28 @@ date: 2018-12-29T11:02:05+06:00
...
+<p class="command"><a href="https://github.com/OSGeo/grass-addons"> $git clone https://github.com/OSGeo/grass-addons </a></p>
 
+To manually compile individual extensions see the [compile and install wiki page](https://grasswiki.osgeo.org/wiki/Compile_and_Install#Addons).
```
Comment (neteler) [inline content/download/addons.en.md:31]:
```suggestion
To manually compile individual addons see the [compile and install wiki page](https://grasswiki.osgeo.org/wiki/Compile_and_Install#Addons). 
```

File: themes/grass/layouts/download/addons.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+</section>
+
+<!-- details page -->
+<section class="single section bg-gray pb-0">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12">
+      <div class="p-5 bg-white">
+	<img src="{{.Site.BaseURL}}/images/logos/grass-Addons.jpg" class="pull-right">
+	<h2 class="page-title">GRASS GIS AddOns</h2>
```
Comment (neteler) [inline themes/grass/layouts/download/addons.html:32]:
```suggestion
	<h2 class="page-title">GRASS GIS Addons</h2>
```

File: themes/grass/layouts/download/list.html [hunk 5]
```diff
@@ -78,7 +78,7 @@ <h3 class="mt-2 mb-2">GRASS GIS AddOns</h3>
 
               <div class="section bg-white mb-4">
                 <div class="row nm">
-                  <div class="col-4 text-center"><img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/grass-SampleData.jpg"></div>
+                  <div class="col-4 text-center"><img alt="GRASS GIS" src="{{.Site.BaseURL}}/images/logos/grass-SampleData.jpg"></div>
                   <div class="col-6">
 		    <h3 class="mt-2 mb-2">GRASS GIS sample data</h3>
 		     <p>Download ready-to-use GRASS GIS sample datasets.</p>
```
Comment (neteler) [inline themes/grass/layouts/download/list.html:84]:
```suggestion
		     <p>Download ready-to-use GRASS GIS sample datasets</p>
```

File: content/download/addons.en.md [hunk 6]
```diff
@@ -1,30 +1,31 @@
...
-You can also use the command line
+You can also use the command line, for example:
 
     g.extension extension=r.fuzzy.system
 
 
 ### <a name="Power-user"></a>Power users/developers
 
-The AddOns source code is hosted in [GRASS-AddOns Github repository](https://github.com/osgeo/grass-addons). To manually compile see the [compile and install wiki page](https://grasswiki.osgeo.org/wiki/Compile_and_Install#Addons). 
+The AddOns source code is also hosted in <a href="https://github.com/OSGeo/grass-addons" target="_blank"><i class="fa fa-github"></i> Github</a>:
```
Comment (veroandreo) [inline content/download/addons.en.md:28]:
```suggestion
The Addons source code is also hosted in <a href="https://github.com/OSGeo/grass-addons" target="_blank"><i class="fa fa-github"></i> Github</a>:
```

**Pull request merged**